WaterLog_Backend/appsettings.json

```diff
@@ -8,6 +8,7 @@
 
     "LiveConnectionString": "Data Source=.\\SQLEXPRESS;Initial Catalog=iot;Integrated Security=true",
     "LocalLiveDBConnectionString": "Server=dev.retrotest.co.za;Database=iot;User Id=group1;Password=fNX^r+UKy3@CtYh5",
-    "LocalConnectionString": "Server=localhost;Database=waterlog;User Id=test;Password=test123"
+  "LocalConnectionString": "Server=localhost;Database=waterlog;User Id=test;Password=test123",
```

Comment:
Storing your passwords hashed saves you from the damages of an unintended individual.
It's like leaving your banking card and your pin on a random table.
At home you might be safe but in a coffee shop, you might soon regret it once unintended people have them in their hands.

Reply:
How would you recommend i go about it?

Reply:
we had an issue with user secrets, they seem to disappear when we deploy on azure devops for of CI's.

Reply:
You can have integrated security as an alternative.
That is SSPI (Security Support Provider Interface).

Reply:
Another thing you can look at is environment variables:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/environments?view=aspnetcore-2.2

Reply:
For azure 
https://docs.microsoft.com/en-us/azure/app-service/web-sites-configure